f/assignGear/f_assignInsignia.sqf

```diff
@@ -46,12 +46,12 @@ private _roleBadge = switch (_typeofUnit) do
 	{
 		switch (_faction) do
 		{
-			case "blu_f" : {_NATO_Medic_Badge};
-			case "opf_f" : {_CSAT_Medic_Badge};
-			case "ind_f" : {"AAF_Medic_Badge"};
-			case "ind_e_f" : {"AAF_Medic_Badge"};
-			case "opf_r_f" : {"CSAT_Pacific_Medic_Badge"};
-			case "blu_w_f" : {"NATO_Pacific_Medic_Bade"};
+			case ("blu_f" or "nato") : {_NATO_Medic_Badge};
```

Comment:
```sqf
case ("blu_f" or "nato") : {_NATO_Medic_Badge};
```
I believe you have to rewrite these to:
```sqf
case ("blu_f");
case ("nato") : {_NATO_Medic_Badge};
```

Or alternatively
```sqf
case (_faction in ["blu_f", "nato"]) : {_NATO_Medic_Badge};
```
but I'm not sure if youd have to use `switch (true) do` in that case.

Reply:
Hmm. I asked in the Arma Discord and one of the pro scripters there told me the format I used was acceptable.

Apparently it isn't though, because the version I've used generates script errors. So that's good.